Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+
+enum MenuOption
```

Comment:
Move this enum code to a separate cs file.

Reply:
Done